. Do not include external vcg headers here
. Do not hardcode include path in the source (and prefix header name with vcg)

--- plystuff.h~	2010-08-24 15:29:55.547211698 +0200
+++ plystuff.h	2010-08-24 15:29:59.650211922 +0200
@@ -61,8 +61,7 @@
 #include <io.h>
 #endif
 
-#include <vcg/space/box3.h>
-#include <wrap/ply/plylib.h>
+#include "vcg-plylib.h"
 using namespace vcg;
 
 #ifdef WIN32
